_includes/cards/choose-card.html

```diff
@@ -5,9 +5,12 @@
     <div class="h2 card-title text-uppercase d-flex align-items-center">
       {{ include.heading }}
     </div>
-		<div class="card-description">
+		<div class="card-description d-none d-md-block">
 			{{ include.description }}
 		</div>
+    <div class="d-block d-md-none">
+			<p class="mt-4"><em>Click to choose</em></p>
+		</div>
```

Comment:
suggestion to get mobile users to click the cards. Some folks whom reviewed were confused.